File: ROID-Forum-Server/ROID-Forum-Server/Convert Server Code/GMGServer/bin/Server/Account Controller/Account.dart
library Account;

class Account {
  String name = "";
  String password = "";
  String email = "";
  String avatarURL = "";
  Account(Map m) {
    name = m['Name'];
    password = m['Password'];
    email = m['Email'];
    if (m.containsKey('AvatarURL')) {
      avatarURL = m['AvatarURL'];
    }
  }
  Map toJSON() {
    Map m = new Map();
    m['Name'] = name;
    m['Password'] = password;
    m['Email'] = email;
    m['AvatarURL'] = avatarURL;
    return m;
  }
}

File: ROID-Forum-Server/ROID-Forum-Server/Account Controller/Account.cs
using System;
using System.Collections.Generic;
namespace ROIDForumServer
{
    public class Account
    {
        public String name = "";
        public String password = "";
        public String email = "";
        public String avatarURL = "";
        public Account(Dictionary<string, object> m)
        {
            name = (string)m.GetValueOrDefault("Name");
            password = (string)m.GetValueOrDefault("Password");
            email = (string)m.GetValueOrDefault("Email");
            if (m.ContainsKey("AvatarURL"))
            {
                avatarURL = (string)m.GetValueOrDefault("AvatarURL");
            }
        }
    }
}